catalog/core/catalog-core-metacardtypes/pom.xml

```diff
@@ -62,6 +62,9 @@
                             catalog-core-api-impl;groupId=org.codice.alliance.catalog.core;version=${project.version},
                             platform-util
                         </Embed-Dependency>
+                        <Private-Package>
+                            ddf.catalog.data.impl.*
```

Comment:
If you are going to private package, then you should remove the embed. Best to use one or the other (otherwise you can end up with multiple instances of the class on the classpath)